nginx/public/node/frontend/public/js/app/manuscript-detail/manuscript-info.template.html

```diff
@@ -13,4 +13,24 @@ <h3 class="h4"><%- name %></h3>
         <dd><pre class="preformatted-text"><%= description %></pre></dd>
         <% } %>
     </dl>
+
+    <% if (imageAttribution || imageLicence || imageLogoUrl) { %>
+        <h3 class="h4">Image Rights</h3>
+        <% if (imageAttribution ) { %>
+            <div>
+            <%= imageAttribution %></dd>
+            </div>
+        <% } %>
+        <% if (imageLogoUrl) { %>
+            <div>
+            <img src="<%= imageLogoUrl %>" alt="Logo of the image source institution" style="max-width: 300px; max-height: 100px; margin-top: 8px; margin-bottom: 8px;"/>
+            </div>
+        <% } %>
+        <% if (imageLicence) { %>
+            <div>
+            <a href="<%= imageLicence %>" target="_blank">Click for licence information.</a>
+        </div>
+        <% } %>
+    <% } %>
```

Comment:
Can you attend to the indentation in this section? Some child elements are at the same level of indentation as their parents, while other paired open/close tags are at different levels of indentation...